src/iocore/net/ConnectionTracker.cc

```diff
@@ -139,37 +143,54 @@ Config_Update_Conntrack_Alert_Delay(const char *name, RecDataT dtype, RecData da
 } // namespace
 
 void
-OutboundConnTrack::config_init(GlobalConfig *global, TxnConfig *txn)
+ConnectionTracker::config_init(GlobalConfig *global, TxnConfig *txn)
 {
   _global_config = global; // remember this for later retrieval.
                            // Per transaction lookup must be done at call time because it changes.
 
-  Enable_Config_Var(CONFIG_VAR_MIN, &Config_Update_Conntrack_Min, txn);
-  Enable_Config_Var(CONFIG_VAR_MAX, &Config_Update_Conntrack_Max, txn);
-  Enable_Config_Var(CONFIG_VAR_MATCH, &Config_Update_Conntrack_Match, txn);
-  Enable_Config_Var(CONFIG_VAR_ALERT_DELAY, &Config_Update_Conntrack_Alert_Delay, global);
+  Enable_Config_Var(CONFIG_SERVER_VAR_MIN, &Config_Update_Conntrack_Min, txn);
+  Enable_Config_Var(CONFIG_SERVER_VAR_MAX, &Config_Update_Conntrack_Max, txn);
+  Enable_Config_Var(CONFIG_SERVER_VAR_MATCH, &Config_Update_Conntrack_Match, txn);
+  Enable_Config_Var(CONFIG_SERVER_VAR_ALERT_DELAY, &Config_Update_Conntrack_Alert_Delay, global);
+}
+
+ConnectionTracker::TxnState
+ConnectionTracker::obtain_inbound(IpEndpoint const &addr)
```

Comment:
I'd like to decouple new code for inbound and keep this PR simple cleanups. 
Especially, this function is already different from PR#10688. 

https://github.com/apache/trafficserver/pull/10688/files#diff-cd94fd94044d4dc1e512c649fb2ce4ef5ed623afff4c365433d32269690bfd90R165-R183

Reply:
Right, that makes sense. I teased this PR out after doing them as one PR and I didn't remove everything perfectly from this base one.

I've updated the PR now to not include obtain_inbound and I reverted some of the comment changes too.  I skimmed through the ConnectionTracker.{cc,h} files and I think I got everything that was "new" out of it, but let me know if I missed anything.

Thanks for reviewing!

Reply:
Thanks for addressing it.